### PR TITLE
Upgrade to Django 2.2

### PR DESCRIPTION
**Pull request by pandabearcoder**
- Django version updated to 2.2 (LTS)
- Updated packages' version
- Used django-markdownx since django-markdown is unsupported
- Use of new url resolver `path`
- Update whitenoise integration with Django. Use of whitenoise via wsgi is deprecated. Middleware is the supported integration

### CODE REVIEW

**Comment by zorexsalvo** [comment]
Thanks @pandabearcoder :100:

File: pythonph/settings.py [hunk 1]
```diff
@@ -43,12 +44,16 @@
 WSGI_APPLICATION = 'pythonph.wsgi.application'
 
 DATABASES = {
+    # 'default': {
+    #     'ENGINE': 'django.db.backends.postgresql_psycopg2',
+    #     'HOST': os.environ.get('POSTGRES_HOST', 'localhost'),
+    #     'NAME': os.environ['POSTGRES_USER'],
+    #     'USER': os.environ['POSTGRES_USER'],
+    #     'PASSWORD': os.environ['POSTGRES_PASSWORD'],
+    # }
```
Comment (zorexsalvo) [inline pythonph/settings.py]: Hi @pandabearcoder you forgot to uncomment the database config.

Comment (pandabearcoder) [reply pythonph/settings.py]: Didn't realized I committed that. Updated!

**Comment by zorexsalvo** [comment]
Hello @pandabearcoder, do you have changes in the Dockerfile/docker-compose.yml's? If yes you can commit those as well.

**Comment by zorexsalvo** [comment]
LGTM :+1:

**Pull request merged**